examples/groups/subgroup.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: groups.gitlab.crossplane.io/v1alpha1
+kind: Group
+metadata:
+  name: example-subgroup

```

Comment:
Since this is just a `group`, should we really add it as another example? My preference is to omit this example

Reply:
I can, but need to add this commented text 
parentIdRef:
      name: example-group
OR
namespaceIdRef:
      name: example-project
to group or project

Reply:
Removed examples/groups/subgroup.yaml